src/components/card-link/index.tsx

```diff
@@ -41,6 +43,28 @@ const CardLink = ({
 				{/* eslint-disable-next-line jsx-a11y/anchor-has-content */}
 				<a aria-label={ariaLabel} className={s.anchor} target={target} />
 			</Link>
+			{children ? (
+				children
+			) : (
+				<div className={s.contentWrapper}>
```

Comment:
💭  Thought: what if, instead of having this component accept _either_ `children` or the other set of "content" props, we create a new component (maybe `CardLinkStructured` or something) that accepts the new set of structured props, and passes them as children to `CardLink`?

Intent would be to retain the "base" `CardLink` component (for those cases where we need full composition), while keeping the door open to build other structured card types (without adding more conditionals within `CardLink` ☺️ )

Reply:
Chatted about this via Slack, and went with a different approach than original, so going to resolve this convo! 😊